### PR TITLE
Updated README and docs with ESM-1b and ESM-2 separately

### PR DESCRIPTION
**Pull request by tomsercu**
# Improve ESM documentation
We split out the papers for ESM-1b and ESM-2 separately, and improve the documentation of esm model_doc.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?
@Rocketknight1

### CODE REVIEW

File: README_zh-hans.md [hunk 1]
```diff
@@ -274,7 +274,9 @@ conda install -c huggingface transformers
...
+1. **[ESM-1b](https://huggingface.co/docs/transformers/main/model_doc/esm)** (from Meta AI) released with the paper [Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences](https://www.pnas.org/content/118/15/e2016239118) by Alexander Rives, Joshua Meier, Tom Sercu, Siddharth Goyal, Zeming Lin, Jason Liu, Demi Guo, Myle Ott, C. Lawrence Zitnick, Jerry Ma, and Rob Fergus.
+1. **[ESM-2](https://huggingface.co/docs/transformers/main/model_doc/esm)** (from Meta AI) released with the paper [Language models of protein sequences at the scale of evolution enable accurate structure prediction](https://doi.org/10.1101/2022.07.20.500902) by Zeming Lin, Halil Akin, Roshan Rao, Brian Hie, Zhongkai Zhu, Wenting Lu, Allan dos Santos Costa, Maryam Fazel-Zarandi, Tom Sercu, Sal Candido, Alexander Rives
+TODO translate
```
Comment (tomsercu) [inline README_zh-hans.md:279]: Note the TODO tag to translate in Chinese :)

**Comment by Rocketknight1** [comment]
Hi @tomsercu! I checked with other team members and we prefer to have only one entry in the list per model class (which would include ESM-1b as well as ESM-2), but it's totally okay to have a longer entry that includes the citations for ESM-1b and ESM-2, and optionally ESM-1v as well.

**Comment by Rocketknight1** [comment]
@tomsercu Don't worry, we have some scripts to handle the copies, so you only have to worry about the English one! I'll run them now and see if I can get these tests to pass.

**Comment by Rocketknight1** [comment]
Remaining issues seem to just be from the specific commit this was branched from - will merge into my PR and hopefully they'll go away. Thanks for this!

**Pull request merged**